build/xtask/src/config.rs

```diff
@@ -287,6 +288,81 @@ impl Config {
             })
             .collect()
     }
+
+    /// Calculates the output region which contains the given address
+    pub fn output_region(&self, vaddr: u64) -> Option<&str> {
+        let vaddr: u32 = match vaddr.try_into() {
```

Comment:
```rust
let vaddr = u32::try_from(vaddr).ok()?;
```

...or use `try_into` if you prefer it, it just takes more typing -- the `.ok()?` is the important bit.

Reply:
Thanks, I always forget that `?` works on `Option` as well!